### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by MaratulyTemirbolat**

### CODE REVIEW

File: apps/blog/templates/blog/login.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <div style="width:50%; margin-left:25%">
+        <h1> Форма авторизации пользователя </h1>
+        <form method = 'post'>
+            {% csrf_token %}
+            {{ form.as_p }}
+            <button type="submit"> Login </button>
+        </form>
+    </div>
+    <br>
+{% endblock %}
```
Comment (inmate37) [inline apps/blog/templates/blog/login.html]: empty line at the end of the file req

File: apps/blog/views.py [hunk 2]
```diff
@@ -11,47 +19,91 @@
...
     paginate_by = 2
     allow_empty = False
     # Чтобы при запросе несуществующей категории была ошибка 404 если пусто
 
-    def get_queryset(self) -> QuerySet:
+    def get_queryset(self) -> QuerySet:  # noqa
         return Post.objects.filter(category__slug=self.kwargs['slug'])
 
-    def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
+    def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:  # noqa
```
Comment (inmate37) [inline apps/blog/views.py:82]: **kwargs: dict[str, Any]

File: apps/abstracts/models.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    datetime_updated = models.DateTimeField(
+        verbose_name="Время обновления",
+        auto_now=True
+    )
+    datetime_deleted = models.DateTimeField(
+        verbose_name="Время удаления",
+        null=True,
+        blank=True
+    )
+    is_deleted = models.BooleanField(
```
Comment (inmate37) [inline apps/abstracts/models.py:18]: do we need "is_deleted" since "datetime_deleted" is presented

Comment (MaratulyTemirbolat) [reply apps/abstracts/models.py:18]: It was provided in order to obtain an opportunity with list_filter for deleted items

Comment (inmate37) [reply apps/abstracts/models.py:18]: Could we use `.filter(datetime_deleted__isnull=True)` for such filtering instead of adding additional boolean field ?

**Pull request merged**